data/governments.txt

```diff
@@ -381,239 +309,43 @@ government "Tehru Miitr"
 
-#color "government: Ultaka" 0.91 0.42 0.09 #for 0.9.17
+color "governments: Ultaka" 0.91 0.42 0.09
 
 government Ultaka
-	color 0.91000003 0.41999999 0.090000004
+	color "governments: Ultaka"
 	"player reputation" -1000
 	"crew attack" 100
 	"crew defense" 100
 	"attitude toward"
-		Coalition -0.01
-		Heliarch -0.01
-		"Indigenous Lifeform" -0.01
-		Drak -0.01
-		Hai -0.01
-		"Navy (Oathkeeper)" -0.01
-		"Hai (Unfettered)" -0.01
-		"Free Worlds" -0.01
-		Wanderer -0.01
-		Pug -0.1
-		Quarg -0.01
-		"Ka'het (Infighting)" -0.01
-		"Kor Efret" -0.01
-		Korath -0.01
-		Republic -0.01
-		Ka'het -0.01
-		Independent -0.01
-		"Pug (Wanderer)" -0.01
-		"Deep Security" -0.01
-		Merchant -0.01
-		Pirate -0.01
-		"Drak (Hostile)" -0.01
-		"Navy Intelligence" -0.01
-		Syndicate -0.01
-		"Kor Mereti" -0.01
-		"Kor Sestor" -0.01
-		Sankrit -0.01
-		Altera -0.01
-		"Dels Ufulre" -0.01
-		ZIS -0.2
-		"Dels Dardornai" -0.01
-		"Erader Darua" -0.01
-		Dels -0.01
-		Donko -0.01
-		"Dels Kaote" -0.01
-		Erm -0.01
-		Geeva -0.01
-		Nuru -0.01
-		Ignaggen -0.01
-		Qanse -0.01
-		Wallagi -0.01
-		Resmakia -0.01
-		Revos -0.1
-		"Kroom Kanai" -0.01
-		"Kroom Kakaima" -0.01
-		"Erader Narpul" -0.01
-		"Qoro Siogal" -0.01
-		"Erader Mukube" -0.01
-		Kumakk -0.01
-		"Erader Kasiva" -0.01
-		"Makau A Qoro" -0.01
-		Taroun -0.2
-		"Tehru Insker" -0.01
-		Zyr'c -0.01
-		"Your Worlds" -0.01
-		Exporiane -0.01
-		Ultaka(Alrul) 10
-		"Vuskr Zorcn" -0.01
-		Vareen -0.01
-		Kaltheim -0.01
-		"Pug (Hostile)" -0.01
-		Aumar -0.01
-		Ixkek -0.01
-		Mil'taj -0.01
-		"Tehru Miitr" -0.01
-		Zorcn -0.01
-		Tuk'ux -0.01
-		Ultaka(Mega) 10
-		"Sayari Plushie" -0.01
-		"Yuyuko Saigyouji" -0.01
-		"Illumina Mining Company" -0.01
-		"Rogue" -0.01
-		"Akralva" -0.01
-		"Ceasrilis" -0.01
-		"SpaceCrate" -.01
-		Athmagr -.01
-		"Bounty Hunter that Won't Enter Hai Space" -.01
-		"Bounty Hunter that Attacks Hai" -.01
-		"Elenctic Commune" -.01
-		"Hai (Friendly Unfettered)" -.01
-		"Pirate (Devil-Run Gang)" -.01
-		"Smuggler (Hai Trafficker)" -.01
-		"Forest (Predator)" -.01
-		"Free Worlds that won't enter wormhole" -.01
-		"Gegno" -.01
-		"Gegno Scin" -.01
-		"Gegno Vi" -.01
-		"Gegno Scin (Neutral)" -.01
-		"Gegno Vi (Neutral)" -.01
-		"Gegno Vi (Duelist A)" -.01
-		"Gegno Vi (Duelist B)" -.01
-		"Hai (Wormhole Access)" -.01
-		"Hai Merchant" -.01
-		"Hai Merchant (Sympathizers)" -.01
-		"Hai Merchant (Human)" -.01
-		"Hai (Friendly Unfettered)" -.01
-		"Ka'sei" -.01
-		"Korath (Civilian)" -.01
-		"Parrot" -.01
-		"Pirate (Rival)" -.01
-		"Remnant (Research)" -.01
-		"Republic that won't enter wormhole" -.01
-		"Republic (Friendly)" -.01
-		"Syndicate that won't enter wormhole" -.01 #pls just make gov variant.
-		"vyu-Rulei" -.01
-		"Rulei" -.01
+		"Pug" -0.1
+		"Revos" -0.1
+		"Taroun" -0.2
+		"Ultaka(Alrul)" 10
+		"Ultaka(Mega)" 10
+		"ZIS" -0.2
+	"default attitude" -0.01
 	language Ultaka
 	"bribe" 0
 
-government Ultaka(Mega)
+government "Ultaka (Mega)"
+	"display name" "Ultaka"
```

Comment:
```suggestion
government "Ultaka(Mega)"
	"display name" "Ultaka (Mega)"
```
This is what I meant. Don't change the government name, change just the display name. Basically, don't change any of the "ID" / internal name referred by other nodes and savefile, it can and will cause problems and incompatibilities.